tests/encfs/primary.py

```diff
@@ -0,0 +1,30 @@
+import os
+from flask import Flask, request, Response
+
+app = Flask(__name__)
+
+ENCFS_MOUNT=os.environ["ENCFS_MOUNT"]
+
+@app.route('/read_file', methods=['GET'])
+def read_file():
+
+    file_path = os.path.join(ENCFS_MOUNT, request.args.get("path"))
+    if not os.path.exists(file_path):
+        return {"error": "File not found"}, 404
+
+    with open(file_path) as f:
+        return f.read(), 200
+
+@app.route('/write_file', methods=['POST'])
+def write_file():
+
+    file_path = os.path.join(ENCFS_MOUNT, request.args.get("path"))
+    try:
+        with open(file_path, "w") as f:
+            f.write(request.data.decode())
+        return f'{request.args.get("path")} written to', 200
+    except OSError as e:
+        return e.strerror, 400
```

Comment:
nit: maybe status code 500 would be more appropriate?

Reply:
Probably, I used 400 to match the version in the examples repo, @hgarvison was there a particular reason to use 400 over 500?

Reply:
I think you did the initial error code definitions in the HTTP servers for the tests and if I made the tests, I just copied the formatting. So no special reasoning from me.

Reply:
Haha I have the memory of a sieve, great if we're all happy with 500 I'll change it